src/test/app/forms/validation/validators/isLessThan.ts

```diff
@@ -0,0 +1,46 @@
+import { expect } from 'chai'
+import { IsLessThanConstraint } from 'forms/validation/validators/isLessThan'
+import { ValidationArguments } from 'class-validator'
+
+const DEFAULT_VALUE = 200
+
+function validationArgs (value: number): ValidationArguments {
+  return {
+    value: undefined,
+    targetName: undefined,
+    object: { relatedField: value },
+    property: undefined,
+    constraints: ['relatedField']
+  }
+}
+
+describe('IsLessThanConstraint', () => {
+  const constraint: IsLessThanConstraint = new IsLessThanConstraint()
+
+  describe('validate', () => {
+
+    describe('should return true when ', () => {
+      it('given an undefined value', () => {
+        expect(constraint.validate(undefined)).to.equal(true)
+      })
+
+      it('given a value less than relatedProperty', () => {
+        expect(constraint.validate(10, validationArgs(DEFAULT_VALUE))).to.equal(true)
+      })
+    })
+
+    describe('should return false when ', () => {
+      it('given an undefined related property', () => {
```

Comment:
I think this case is more of an application error than user input error and should probably throw an error instead?

Reply:
i just reverted what we used to have. IsLessThan was remove couple of days ago, as we implemented isLessThanSumOf

Reply:
I can remove it